### PR TITLE
Add handle Niels

### PR DESCRIPTION
**Pull request by JCZuurmond**
### Checklist

- [x] When adding a Github handle to be tracked that is not yours, request the user to approve the PR

### CODE REVIEW

**Review by NielsZeilemaker — DISMISSED** [review]

**Comment by JCZuurmond** [comment]
@NielsZeilemaker : Could you approve once more? Had to solve the merge conflict

**Review by NielsZeilemaker — APPROVED** [review]

**Pull request merged**
